src/visualizers/ShiftCompare.ts

```diff
@@ -117,7 +116,6 @@ class VizShiftCompare extends P5Visualizer {
 }
 
 export const exportModule = new VisualizerExportModule(
-    'Shift Compare',
-    VizShiftCompare,
-    ''
+    ShiftCompare,
+    'A grid showing pairwise congruence of sequence entries, to some modulus'
```

Comment:
whether or not they have a period should probably be consistent.  right now some do, some don't